### PR TITLE
Update CSI - White Paper - Introduction to CS Internship --fa-ir.md

### PR DESCRIPTION
**Pull request by zahra-ahangari**
- Fix image issues
- Correct file name

### CODE REVIEW

**Comment by zahra-ahangari** [comment]
I fixed the issues you reported. It can be merged after your approval. @mehrandvd

**Comment by mehrandvd** [comment]
It's OK with me.

**Review by mehrandvd — APPROVED** [review]

**Pull request merged**
